src/rules/no-important.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * @fileoverview Rule to prevent !important in CSS.
+ * @author Yann Bertrand
+ */
+
+//-----------------------------------------------------------------------------
+// Rule Definition
+//-----------------------------------------------------------------------------
+
+export default {
+	meta: {
+		type: /** @type {const} */ ("problem"),
+
+		docs: {
+			description: "Disallow important annotations",
+			recommended: true,
+		},
+
+		messages: {
+			forbiddenImportantUsage: "!important annotations are not allowed.",
```

Comment:
Can we shorten this to `unexpectedImportant`?

And we generally like to be a bit gentler with the messages:

```suggestion
			unexpectedImportant: "Unexpected !important annotation found.",
```

Reply:
Done. Thanks for the suggestions